gitlab/base.py

```diff
@@ -150,6 +150,10 @@ def _create_managers(self) -> None:
         # annotations. If an attribute is annotated as being a *Manager type
         # then we create the manager and assign it to the attribute.
         for attr, annotation in sorted(self.__annotations__.items()):
+            # We ignore creating a manager for the 'manager' attribute as that
+            # is done in the self.__init__() method
+            if attr in ("manager",):
+                continue
```

Comment:
Just wondering is there's any specific reason this isn't just `if attr == "manager":`?

Reply:
For the future! Besides that no good reason.